test/attributes.js

```diff
@@ -62,15 +74,11 @@ describe('attributes', function() {
   });
   describe('Object.prototype property', function() {
     it('is not considered as a boolean attribute and shouldn\'t be omitted', function() {
```

Comment:
For the purpose of just checking Object.prototype properties, it's enough to test for the `constructor` property. Attributes like `valueOf` are invalid since they contain an uppercase letter.
Requesting @fix-fix to review this since they wrote this test.